### PR TITLE
refactor(run-protocol): rename ReallocateReward to reallocateWithFee

### PR DESCRIPTION
**Pull request by dtribble**
refs: #4415

## Description

Naming improvements to better follow the logic of reward fees in vaults.

### Security Considerations

Pure refactoring. There should be no security impact.

### Documentation Considerations

These are mostly internal but there will be external facing changes (e.g., fee pool rather than reward pool).

### Testing Considerations

Test and style automation should be sufficient as these are pure refactors.

### CODE REVIEW

File: packages/run-protocol/src/vaultFactory/vault.js [hunk 1]
```diff
@@ -606,11 +606,12 @@ export const makeInnerVault = (
     }
 
     // mint to vaultSeat, then reallocate to reward and client, then burn from
-    // vaultSeat. Would using a separate seat clarify the accounting?
+    // vaultSeat. Would using a separate seat clarify the accounting?\
```
Comment (turadg) [inline packages/run-protocol/src/vaultFactory/vault.js]:
```suggestion
    // vaultSeat. Would using a separate seat clarify the accounting?
```

Comment (Chris-Hibbert) [inline packages/run-protocol/src/vaultFactory/vault.js]:
```suggestion
    // vaultSeat. Would using a separate seat clarify the accounting?
```

**Review by turadg — APPROVED** [review]
Change looks fine (aside from comment typo).

Description says "Naming improvements" but I only see one.

If it's just one I'd suggest retitling the PR,
```
refactor(run-protocol): rename ReallocateReward to reallocateWithFee
```
and then merge by squashing.

**Review by Chris-Hibbert — APPROVED** [review]
one tiny typo.

**Pull request merged**